src/System.Private.CoreLib/shared/System/Array.cs

```diff
@@ -1739,8 +1739,8 @@ private int PickPivotAndPartition(int lo, int hi)
 
                 while (left < right)
                 {
-                    while (comparer.Compare(keys[++left], pivot) < 0) ;
-                    while (comparer.Compare(pivot, keys[--right]) < 0) ;
+                    while (comparer.Compare(keys[++left], pivot) < 0);
+                    while (comparer.Compare(pivot, keys[--right]) < 0);
```

Comment:
VS will re-add the space here if you auto format (or is there a setting around this?)

Reply:
Yeah, I was hoping there was an .editorconfig setting for it, but there wasn't.  I've reverted those spacing changes to prevent a ping-pong fight between StyleCop and VS; we just won't turn on that StyleCop rule in the build.